atc/api/handler.go

```diff
@@ -1,6 +1,7 @@
 package api
 
 import (
+	"github.com/concourse/concourse/atc/api/wallserver"
```

Comment:
nit: we tend to separate the imports, keeping standard lib ones on the top, and                                                                                                      [0/1]
all-else in the bottom :grin:


```diff
diff --git a/atc/api/handler.go b/atc/api/handler.go
index 3be7caaa6..dc12a0a95 100644
--- a/atc/api/handler.go
+++ b/atc/api/handler.go
@@ -1,12 +1,9 @@
 package api

 import (
-       "github.com/concourse/concourse/atc/api/wallserver"
        "net/http"
        "path/filepath"

-       "github.com/concourse/concourse/atc/api/usersserver"
-
        "code.cloudfoundry.org/lager"
        "github.com/concourse/concourse/atc"
        "github.com/concourse/concourse/atc/api/artifactserver"
@@ -23,7 +20,9 @@ import (
        "github.com/concourse/concourse/atc/api/resourceserver"
        "github.com/concourse/concourse/atc/api/resourceserver/versionserver"
        "github.com/concourse/concourse/atc/api/teamserver"
+       "github.com/concourse/concourse/atc/api/usersserver"
        "github.com/concourse/concourse/atc/api/volumeserver"
+       "github.com/concourse/concourse/atc/api/wallserver"
        "github.com/concourse/concourse/atc/api/workerserver"
        "github.com/concourse/concourse/atc/creds"
        "github.com/concourse/concourse/atc/db"
```diff


e.g.: https://github.com/concourse/concourse/pull/4825/commits/c757369352f9bf1e79d9567f2e36611c7af27c97